### PR TITLE
[Workers] Runtime API nav revision

### PR DESCRIPTION
**Pull request by deadlypants1973**
**BEFORE COMMENTING ON THIS PR**, refer to PCX-6575 for a report on industry comparison, design principles and user research for this change.

Related to #9592

### CODE REVIEW

File: content/workers/runtime-apis/streams/_index.md [hunk 1]
```diff
@@ -1,8 +1,100 @@
...
+
+Minimizing buffering is especially important for processing or transforming response bodies larger than the Worker's memory limit. For these cases, streaming is the only implementation strategy.
+
+{{<Aside type="note">}}
+
+By default, Cloudflare Workers is capable of streaming responses using the [Streams APIs](https://developer.mozilla.org/en-US/docs/Web/API/Streams_API). To maintain the streaming behavior, you should only modify the response body using the methods in the Streams APIs. If your Worker only forwards subrequest responses to the client verbatim without reading their body text, then its body handling is already optimal and you do not have to use these APIs.
+
+{{</Aside>}}
+
+The two primitives developers use to perform active streaming are [`TransformStream`](/workers/runtime-apis/streams/transformstream/) and the [`ReadableStream.pipeTo()`](/workers/runtime-apis/streams/readablestream/#methods) method.
```
Comment (marciocloudflare) [inline content/workers/runtime-apis/streams/_index.md]:
```suggestion
The two primitives developers used to perform active streaming are [`TransformStream`](/workers/runtime-apis/streams/transformstream/) and the [`ReadableStream.pipeTo()`](/workers/runtime-apis/streams/readablestream/#methods) method.
```

File: content/workers/runtime-apis/websockets/_index.md [hunk 2]
```diff
@@ -16,7 +16,9 @@ WebSockets utilize a simple event-based system for receiving and sending message
 
 {{</Aside>}}
 
-## Writing a WebSocket Server
+Learn the basics of WebSockets on Cloudflare Workers, both from the perspective of writing WebSocket servers in your Workers functions, as well as connecting to and working with those WebSocket servers as a client.
+
+## Write a WebSocket Server
```
Comment (marciocloudflare) [inline content/workers/runtime-apis/websockets/_index.md]: Does server need to be capitalized? I don't think it needs, but defer to you

**Review by marciocloudflare — APPROVED** [review]

File: content/workers/runtime-apis/websockets/_index.md (hunk 2 above)
Comment (deadlypants1973) [inline content/workers/runtime-apis/websockets/_index.md]:
```suggestion
## Write a WebSocket server
```

**Comment by deadlypants1973** [comment]
merge conflict resolution by #9861

**Pull request closed without merging**